tests/PullPaymentTest.php

```diff
@@ -121,10 +123,20 @@ public function testAllPayoutMethods(): void
             paymentMethods: ['BTC-LightningNetwork']
         );
 
+        $lightningClient = new \BTCPayServer\Client\LightningInternalNode($this->host, $this->apiKey);
```

Comment:
also here, I merged that in the other PR?

Reply:
i submitted this PR based on the branch from the previous PR instead of the base, extremely dumb, my b

Reply:
even with that it should not be here as it is now already in 3.x, it should show conflict or not show it at all, not sure what happens when I merge that :D will do squash merge and we will see :pray: